[
    {
        "content": "<p>This topic was moved by <span class=\"user-mention silent\" data-user-id=\"234973\">Till Schneidereit</span> to <a class=\"stream-topic\" data-stream-id=\"290350\" href=\"/#narrow/stream/290350-wamr/topic/WAMR.20contributions\">#wamr &gt; WAMR contributions</a></p>",
        "id": 258722695,
        "sender_full_name": "Notification Bot",
        "timestamp": 1634914369
    },
    {
        "content": "<p>(I moved this into the wamr stream for more visibility)</p>",
        "id": 258722729,
        "sender_full_name": "Till Schneidereit",
        "timestamp": 1634914389
    },
    {
        "content": "<p>\"When working with WAMR and exceptions, is there any way to configure a callback (native function) to be triggered by WAMR whenever an exception in the WAMR domain occurs? I found <code>wasm_runtime_set_enlarge_mem_error_callback</code>, so I am looking for something similar but for any type of exception.</p>",
        "id": 509147997,
        "sender_full_name": "Borja Tellado",
        "timestamp": 1743413888
    }
]